torch/profiler/profiler.py

```diff
@@ -326,7 +326,8 @@ def handler_fn(prof) -> None:
                 raise RuntimeError("Can't create directory: " + dir_name) from e
         if not worker_name:
             worker_name = "{}_{}".format(socket.gethostname(), str(os.getpid()))
-        file_name = "{}.{}.pt.trace.json".format(worker_name, int(time.time() * 1000))
+        # Use nanosecond here to avoid naming clash when exporting the trace
+        file_name = "{}.{}.pt.trace.json".format(worker_name, time.time_ns())
```

Comment:
We live in Python-3 world now, why not use f-strings?
```suggestion
        file_name = f"{worker_name}.{time.time_ns()}.pt.trace.json"
```